<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportConsensusWidget</class>
 <widget class="QWidget" name="ExportConsensusWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>215</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="savePathWidget" native="true">
     <layout class="QVBoxLayout" name="savePathLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="settingsWidget" native="true">
     <layout class="QVBoxLayout" name="settingsLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="savePathGroup" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="pathLe"/>
        </item>
        <item>
         <widget class="QToolButton" name="browseBtn">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QComboBox" name="formatCb"/>
      </item>
      <item>
       <widget class="QCheckBox" name="addToProjectChb">
        <property name="text">
         <string>Add to project</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="exportBtn">
        <property name="text">
         <string>Export</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="settingsGroup" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="keepGapsChb">
        <property name="text">
         <string>Keep gaps</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="nonAlphabetSymbolsPolicyGb">
        <property name="title">
         <string>Non-alphabet symbols policy</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="skipRb">
           <property name="text">
            <string>Skip</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="gapRb">
           <property name="text">
            <string>Replace with gaps</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="defaultSymbolRb">
           <property name="text">
            <string>Replace with default symbol</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="keepAllSymbolsRb">
           <property name="text">
            <string>Keep all symbols</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>127</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
